command/certificate/needsRenewal.go

```diff
@@ -0,0 +1,135 @@
+package certificate
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"github.com/smallstep/cli/errs"
+	"github.com/urfave/cli"
+	"io/ioutil"
+	"os"
+	"time"
+)
+
+func needsRenewalCommand() cli.Command {
+	return cli.Command{
+		Name:      "needs-renewal",
+		Action:    cli.ActionFunc(needsRenewalAction),
+		Usage:     `Check if a certificate needs to be renewed`,
+		UsageText: `**step certificate needs-renewal** <crt_file> or <host_name> [**--expires-in <duration>]`,
+		Description: `**step certificate needs-renewal** Checks certificate expiration from file or from a host if 
+		the certificate is over 66% of its lifetime. If the certificate needs renewal this command will return '0'.
+		If the certificate is not far enough into its life, it will return '1'. If validation fails, or if an error occurs, 
+		this command will produce a non-zero return value.
+## POSITIONAL ARGUMENTS
+
+<crt_file>
+: The path to a certificate to validate.
+
+<host_name>
+: Address of remote host 
+
+## EXIT CODES
+
+This command returns 0 if needing renewal, or returns 1 if the certificate does not need renewal. It will return 255 if any errors occurred
+
+## EXAMPLES
+Check certificate for renewal using custom directory 
+'''
+$ step certificate needs-renewal ./certificate.crt 
+'''
+Check if certificate will expire within a given time 
+'''
+$ step certificate needs-renewal ./certificate.crt --expires-in 1h15m
+'''
+`,
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  "expires-in",
```

Comment:
See the description of the `not-before` flag in `command/certificate/create.go`. Use the second part of that description that talks about duration for this flag as well. And additionally, I think we want the option to use a percent for this flag. So if I pass `70%` it should be parsed as a percent. Let's try to make that clear in the description.

So for example -- the beginning of `not-before` description starts with `<time|duration>`. This description could start with `<percent|duration>`.